7_react/react1/week2/todo_list_app/src/components/todoList.jsx

```diff
@@ -0,0 +1,52 @@
+const TodoList = ({ todos, setTodos }) => {
+  const handleComplete = (todo) => {
```

Comment:
I suppose you could have used the same destructuring as you have done in the `handleDelete` method. (i.e. `{ id }`) That would make your code consistent and more elegant. 😄